docs/en/writing-running-appium/tutorial/swipe/ios-mobile-screen.md

```diff
@@ -0,0 +1,92 @@
+## iOS 'mobile:': Screen swipe
+
+XCTest framework supports unique gestures like "mobile:scroll" and
+"mobile:swipe". They are not so flexible as UIScrollable but still
+useful.
+
+As usual swipe performs swipe action while scroll tries to change one
+visible view port only.
+
+!Note! Scroll direction differs between scroll and swipe commands
+
+```java
+/**
+ * Performs screen scroll
+ *
+ * @param dir the direction of scroll
+ * @version java-client: 7.3.0
+ **/
+public void mobileScrollScreenIOS(Direction dir) {
+    System.out.println("mobileScrollScreenIOS(): dir: '" + dir + "'"); // always log your actions
+
+    // Animation default time:
+    //  - iOS: 200 ms
+    // final value depends on your app and could be greater
+    final int ANIMATION_TIME = 200; // ms
+    final HashMap<String, String> scrollObject = new HashMap<String, String>();
```

Comment:
```
 driver.executeScript("mobile:scroll", ImmutableMap.of("direction", dir.name().toLowerCase()))
```

Reply:
it will not work like this. beginner need to see and feel how hashMap builds and see that scroll direction between swipe and scroll are different.